Facil/ex04.js

```diff
@@ -0,0 +1,6 @@
+//4 – Percorra cada item do array de vegetais, utilizando o for, e exiba o seu índice e o valor.
+
+let vegetais = ["Repolho", "Nabo", "Rabanete", "Cenoura"];
+let result = vegetais.forEach((item, indice) => {
```

Comment:
Utilizar o laço de repetição **for**. https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Statements/for